chsdi/__init__.py

```diff
@@ -22,6 +22,27 @@ def cleanup(request):
     return session
 
 
+def add_cors_route(config, pattern, service, headers=None, methods=[]):
+    # The use of "GET" also implies that the view will respond to "HEAD".
+    allowed_methods = ['OPTIONS', 'HEAD']
+
+    def view(request):  # pragma: no cover
+        response = request.response
+        response.cache_control.no_cache = True
+        response.cache_control.max_age = 0
+        response.headers['Access-Control-Allow-Origin'] = '*'
```

Comment:
I think Origin must be in header, but anyway it's overriden at apache level and we put * everywhere.

To handle CORS properly we should maybe remove some of the stuff here:
https://github.com/geoadmin/mf-chsdi3/blob/master/apache/wsgi.conf.in#L93

and add a subscriber on new response something like:
https://gist.github.com/kamalgill/b1f682dbdc6d6df4d052

Reply:
access control allow methods should be removed from apache configuration as it is incorrect now.
It always sets all the methods.